configs/miniTreeEntries.dat

```diff
@@ -0,0 +1,26 @@
+priorCommitNumberForWhichTheseEventCountNumbersAreValid
+e6114abb9badb083f04b5c9433e721cb1c078b8d
+[?1034h# Dataset miniTree_flavoursideband miniTree_lowdileptonsideband miniTree_signal_ee miniTree_signal_mumu miniTree_dytagandprobe 
```

Comment:
If you manage to understand where this bad thing arises, would be great. The commit part should be preceded by a # at the beginning of the line, such that can be discarded by other scripts